.emacs.d/settings/custom.el

```diff
@@ -8,7 +8,7 @@
 
 ; color theme
 (add-to-list 'custom-theme-load-path (make-plugin-path "color-theme-solarized"))
-(load-theme 'solarized-dark 1)
+(load-theme 'solarized 1)
 (setq solarized-termcolors 256)
```

Comment:
Maybe. I couldn't figure out how to make that work, so I left it at default. A more complete fix would definitely be better. 



> On May 19, 2015, at 2:22 PM, Sagun Khatri <notifications@github.com> wrote:
> 
> In .emacs.d/settings/custom.el:
> 
> > @@ -8,7 +8,7 @@
> >  
> >  ; color theme
> >  (add-to-list 'custom-theme-load-path (make-plugin-path "color-theme-solarized"))
> > -(load-theme 'solarized-dark 1)
> > +(load-theme 'solarized 1)
> >  (setq solarized-termcolors 256)
> Won't you need another line that will set 'dark or 'light theme?
> 
> Something along this line: (set-frame-parameter nil 'background-mode 'dark)?
> 
> —
> Reply to this email directly or view it on GitHub.
>